README.md

```diff
@@ -68,12 +68,32 @@ docker compose --profile all down --volumes
 
 ## Advanced 👷
 
+### Development
 If you are a developer on `bee-api` or `bee-ui` and want to run only the supporting infrastructure,
 use the profile `infra`, e.g.:
 
 ```shell
 docker compose --profile infra up -d
 ```
+### Connecting to host resources
```

Comment:
Please move this to troubleshooting guide. 

The generic comments are not necessary we only need to access host for `ollama`, the guide is organized by error messages, please add it under the specific error message which occurs when you cannot connect to ollama.

You can also improve the error message in ./bee-stack.sh directly